package.json

```diff
@@ -6,7 +6,7 @@
   "repository": "https://github.com/adopteitor/adopteitor-api",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node src/app.js",
+    "start": "nodemon src/app.js",
```

Comment:
I'm not sure if we should use `nodemon` for the start script, since we have it on the `dev` script and nodemon is not meant for production. 

My idea was that `start` would be used to run the service once it's deployed. `dev` uses nodemon and restarts on change for development purposes.

Reply:
You are right. I'm gonna fix this before merging